src/Microsoft.AspNetCore.Mvc.TagHelpers/LinkTagHelper.cs

```diff
@@ -421,7 +421,8 @@ private void BuildLinkTag(string href, TagHelperAttributeList attributes, TagHel
                 }
                 else
                 {
-                    AppendAttribute(attribute.Name, attribute.Value, builder);
+                    attribute.CopyTo(builder);
+                    builder.AppendHtml(" ");
```

Comment:
Let's be consistent and write out the spaces only where they're needed i.e. remove space o/p on [line 407](https://github.com/aspnet/Mvc/pull/4587/files#diff-c665e699fd5f8e3566e73418b930e54fR407) and move this line above [line 416](https://github.com/aspnet/Mvc/pull/4587/files#diff-c665e699fd5f8e3566e73418b930e54fR416).